origin-dapp/src/pages/profile/VerifyAirbnb.js

```diff
@@ -100,7 +100,7 @@ class VerifyAirbnb extends Component {
                 id: 'VerifyAirbnb.placeholderAirbnbProfileUrl',
                 defaultMessage: 'https://www.airbnb.com/users/show/123'
               })}
-              pattern="^https?://www\.airbnb\.com/users/show/\d*$"
+              pattern="^https?:\/\/www\.airbnb\.[a-zA-z.]{2,6}\/users\/show\/\d*$"
```

Comment:
In `[a-zA-z.]`, what is the `.` for? (since `.` matches any non-newline character the other stuff between the brackets is redundant)

also, did you mean `A-z` to be `A-Z` instead?

Reply:
The `.` is for matching a literal `.` inside multi level TLDs, like `.com.au`. Inside a character class like that I don't believe the `.` should be escaped. I'll test again to verify though.

And yep the capitalisation is wrong, thanks for spotting that 😁